.golangci.yml

```diff
@@ -41,6 +42,10 @@ issues:
       # ignore warnings from table tests. https://github.com/kyoh86/scopelint/issues/4
       path: ".*_test.go|store/storetest"
 
+    - linters:
+      - staticcheck
+      path: ".*_test.go|store/storetest"
```

Comment:
I would love to see a couple of HW tickets to add the checks for test files.

Reply:
Totally agree. Let me see if I can spin up some of these.